modules/core/src/test/scala/org/scalasteward/core/forge/ForgeRepoTest.scala

```diff
@@ -0,0 +1,81 @@
+package org.scalasteward.core.forge
+
+import munit.FunSuite
+import org.http4s.Uri
+import org.http4s.implicits._
+import org.scalasteward.core.forge.ForgeType._
+
+/** As much as possible, uris in this test suite should aim to be real, clickable, uris that
+  * actually go to real pages, allowing developers working against this test suite to verify that
+  * the url patterns constructed here actually _are_ valid, and match what forges are currently
+  * using in the real world.
+  */
```

Comment:
**praise:** This is gold!

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>